docs/how_to/read_asdf_file.rst

```diff
@@ -5,30 +5,32 @@ Read an ASDF file into a Map
 ****************************
 
 `ASDF <https://asdf-standard.readthedocs.io/en/latest/>`__ is a modern file format designed to meet the needs of the astronomy community [citation needed].
-It has deep integration with Python, sunpy, and Astropy, as well as implementations in other languages.
-It can be used to store known Python objects in a portable, well defined file format.
-It is primarily useful for storing complex Astropy and sunpy objects in a way that can be loaded back into the same form as they were saved.
-It is designed to be an archive file format, with human readable metadata and a simple on-disk layout.
+It has deep integration with Python, SunPy, and Astropy, as well as implementations in other languages.
+It can be used to store known Python objects in a portable, well-defined file format.
+It is primarily useful for storing complex Astropy and SunPy objects in a way that can be loaded back into the same form as they were saved.
+It is designed to be an archive file format, with human-readable metadata and a simple on-disk layout.
```

Comment:
```suggestion
It has deep integration with Python, sunpy, and Astropy, as well as implementations in other languages.
It can be used to store known Python objects in a portable, well-defined file format.
It is primarily useful for storing complex Astropy and sunpy objects in a way that can be loaded back into the same form as they were saved.
It is designed to be an archive file format, with human-readable metadata and a simple on-disk layout.
```

Reply:
Both uses of SunPy here were deliberate to me, certainly the first one. There's at least DKIST as an affiliated package and soon ndcube which supports asdf.

Reply:
Ok. Ill correct that.